go.mod

```diff
@@ -5,6 +5,7 @@ go 1.14
 replace (
 	github.com/NetEase-Object-Storage/nos-golang-sdk => github.com/karuppiah7890/nos-golang-sdk v0.0.0-20191116042345-0792ba35abcc
 	go.etcd.io/etcd => github.com/eddycjy/etcd v0.5.0-alpha.5.0.20200218102753-4258cdd2efdf
+	github.com/chartmuseum/storage => github.com/farmersedgeinc/storage v0.9.1-0.20200810211930-42f6c50507a1
```

Comment:
Can we remove this? The chartmuseum 'main' branch now has your changes

Reply:
whoops - my apologies, has not been yet

Reply:
just bumping: reviewing the storage library changes will be a blocker for getting this merged. idk when you're planning your next release so maybe that's not an issue yet